tests/metrics/README.md

```diff
@@ -4,19 +4,21 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo -E make metrics-tests
```

Comment:
@jodh-intel it does!!! @chavafg , if we can add that `PATH=$PATH` bit, that'd be good.